models/src/test/scala/coop/rchain/models/rholang/SortTest.scala

```diff
@@ -563,24 +563,13 @@ class ParSortMatcherSpec extends FlatSpec with Matchers {
               )
             )
           ),
-          Connective(VarRefBody(VarRef(0, 2))),
-          Connective(ConnInt(true)),
-          Connective(ConnBool(true)),
-          Connective(ConnString(true)),
-          Connective(ConnByteArray(true)),
-          Connective(ConnUri(true)),
           Connective(ConnNotBody(Par()))
         ),
         connectiveUsed = true
       )
     val sortedParExpr: Par =
       Par(
         connectives = List(
-          Connective(ConnBool(true)),
-          Connective(ConnInt(true)),
-          Connective(ConnString(true)),
-          Connective(ConnUri(true)),
-          Connective(ConnByteArray(true)),
```

Comment:
While I agree those don't fit the test's description, I believe we should still have a test for  the respective order of those connective variants. Please either add it in this class, or try utilize the `assertOrder` in https://github.com/rchain/rchain/blob/4846440c2349cb37f94967dc0846a9fbe22ee85c/rholang/src/test/scala/coop/rchain/rholang/interpreter/SortSpec.scala#L15 (probably extending it to accept varargs)

Reply:
Done